### DISCUSSION TITLE
meson with verbose ninja

### DISCUSSION THREAD

**Discussion by Kreijstal** [mesonbuild/meson#10648] [Q&A] [answered]
ninja is failing saying a subcommand error failed with no error message, so I want to activate verbose ninja, how do I do that?

```
meson compile --ninja-args "-v"
usage: meson compile [-h] [--clean] [-C WD] [-j JOBS] [-l LOAD_AVERAGE] [-v] [--ninja-args NINJA_ARGS] [--vs-args VS_ARGS] [--xcode-args XCODE_ARGS] [TARGET ...]
meson compile: error: argument --ninja-args: expected one argument
```
this doesn't work

**Comment by eli-schwartz** [comment]
I would just use

```
ninja -v
```

rather than trying to pas arguments through *two* argument parsers and running a slow python wrapper on top.

**Comment by pallaswept** [comment, marked as answer]
> '--ninja-args "-v"'

is missing an `=`. use
'--ninja-args="-v"'

hope that helps
